### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by leo8a**
Adds a security response policy. The proposed text aligns with the response of other OpenShift components.

Signed-off-by: Leonardo Ochoa-Aday [lochoa@redhat.com](mailto:lochoa@redhat.com)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Security Response
+If you've found a security issue that you'd like to disclose confidentially please contact
+Red Hat's Product Security team. Details at https://access.redhat.com/security/team/contact
+
+# License
+OKD is licensed under the Apache License, Version 2.0. Some components may be licensed
```
Comment (javipolo) [inline SECURITY.md:6]: should it say `OKD` ?

**Comment by donpenney** [comment]
Just needs a couple of tweaks to satisfy our markdown linter checks

**Comment by leo8a** [comment]
hey @imiller0, after chatting with Don, we realized that maybe the Security Response should be added at the openshift-kni org level, similar to the openshift one -> https://github.com/openshift/.github 

can you help us to confirm this? if so, can we have it also in the openshift-kni org, in this way all repos under this org may automatically be benefited from this.

**Comment by donpenney** [comment]
Closing this PR in favour of pursuing an openshift-kni org response

**Pull request closed without merging**